.travis.yml

```diff
@@ -30,3 +31,6 @@ script:
   - go vet ./management/...
   - test -z "$(golint ./Gododir/... | tee /dev/stderr)"
   - go vet ./Gododir/...
+  - set -e; for v in ./arm/... ./datalake-store/... ./Gododir/... ./management/... ./storage/...; do errcheck -ignoretests "$v"; done
```

Comment:
are you sure `errcheck` returns with non-zero exitcode when it finds issues? if so you may still need the `test -z`.